lib/tX.ml

```diff
@@ -72,25 +72,36 @@ module Request = struct
 end
 
 module Response = struct
-  [%%cenum
   type status =
     | DROPPED [@id 0xfffe]
     | ERROR   [@id 0xffff]
     | OKAY    [@id 0]
     | NULL    [@id 1]
```

Comment:
Maybe remove `[@id 0xfffe]` annotations? Or make them into comments at least.

Reply:
done in 10abed0